instances.go

```diff
@@ -91,6 +106,24 @@ func (c *Client) GetAllowedNetworkInstancePolicy(ctx context.Context) (*Instance
 	return &policyResponse.Policies[0], nil
 }
 
+// GetKeyAccessInstancePolicy retrieves the key create import access policy details associated with the instance.
+// For more information can refer the Key Protect docs in the link below:
+// <kp link>
```

Comment:
Did you want to put tentative links? The current tentative url appears to be `https://cloud.ibm.com/docs/key-protect?topic=key-protect-manage-key-creation-import`